config/webpack.config.prod.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
   output: {
     path: paths.appBuild,
     filename: 'index.js',
-    libraryTarget: 'umd'
+    library: 'VanillaReact',
```

Comment:
I think it should be called `VanillaFrameworkReact` otherwise its a oxymoron being Vanilla and React.